pages/about.html

```diff
@@ -0,0 +1,154 @@
+          <a href="../index.html"
+            ><img src="../Images/logo.png" alt="logo"
+          /></a>
+          <a href="../index.html"><b>World Architectural Summit 2023</b></a>
+        </div>
+
+        <ul class="nav-links">
+          <li><a href="../index.html">Home</a></li>
+          <li><a href="#">About</a></li>
+          <li><a href="#location">Location</a></li>
+          <li><a href="#archives">Archives</a></li>
+        </ul>
+
+        <ul class="nav-links2">
+          <li><a href="../index.html">Home</a></li>
+          <li><a href="#">About</a></li>
+          <li><a href="#location">Location</a></li>
+          <li><a href="#archives">Archives</a></li>
+          <li class="style">WA Summit</li>
+        </ul>
+      </nav>
+    </header>
+
+    <main>
+      <section class="about">
+        <h1 class="about-heading">WORLD ARCHITECTURAL SUMMIT 2023</h1>
+        <p class="about-paragraph">
+          World Architectural Summit will build on the breakout success of the
+          inaugural edition with a captivating and insightful agenda that will
+          be discussed and debated by some of the region's best and brightest
+          minds. The Summit will provide a platform for construction industry
+          stakeholders to discuss a multitude of topics including future city
+          requirements, net zero building design, resilient design strategies,
+          future transport infrastructure, nurturing future talent and much
+          more. Along with a rich agenda, the summit will also provide
+          unparalleled networking opportunities in full compliance with all
+          local COVID-19 related regulations.
+        </p>
+      </section>
+
+      <section class="location" id="location">
+        <h2 class="location-heading">Location</h2>
+        <div class="line"></div>
+        <p class="location-paragraph">
+          World Architectural Summit will take place in <br /><span
+            ><b>THE OBEROI / DUBAI / UAE on 1st JUNE 2023</b></span
+          ><br />
+          in full compliance with all COVID-19 safety guidelines and protocols.
+        </p>
+      </section>
+
+      <section class="past-summit" id="archives">
+        <div class="summit">
+          <h2 class="past-summit-heading">Archives</h2>
+          <div class="line"></div>
+        </div>
+
+        <p class="past">Browse through our previously held summits</p>
+
+        <div class="past-summit-container">
+          <a href="#" class="past-summit-card1">
+            <p class="card-paragraph">
+              <span><b>2018</b></span
+              ><br />World Architectural Summit in Marina Bay Sands, Signapore
+            </p>
+          </a>
+          <a href="#" class="past-summit-card2">
+            <p class="card-paragraph">
+              <span><b>2021</b></span
+              ><br />World Architectural Summit in New York, United States
+            </p>
+          </a>
+        </div>
+      </section>
```

Comment:
- [ ] Kindly consider following the Original Behance design for font-size, font-weight and the font colors in this section. your font-sizes seem bigger than what's in the Original design. also, the paragraph text has lighter color in the Original design so please try fixing it.
- [ ] Also the images are organized in a horizontal row in mobile screen in the original design. but in your design the images in this section are organized vertically in the mobile screen so consider fixing it.
Original Design
![ui](https://user-images.githubusercontent.com/96683843/226004303-d611ab50-6876-4bae-8ec1-0912fe52413e.jpg)